pkg/controller/operators/olm/operator.go

```diff
@@ -948,98 +913,6 @@ func (a *Operator) EnsureCSVMetric() error {
 	return nil
 }
 
-func (a *Operator) syncGCObject(obj interface{}) (syncError error) {
```

Comment:
note to self: events on these objects by definition cannot be triggers for cleanup, so losing the cross-queueing should in theory not make any difference